frontend/components/dashboard/table/DashboardTableValidators.vue

```diff
@@ -58,21 +58,24 @@ const cappedValidators = computed(() =>
 
 <template>
   <div class="validator_column">
-    <div class="validators">
+    <div v-if="cappedValidators.length <= 3" class="validators">
       <template
-        v-for="v in cappedValidators"
-        :key="v"
+        v-for="validator in cappedValidators"
+        :key="validator"
       >
         <BcLink
-          :to="`/validator/${v}`"
+          :to="`/validator/${validator}`"
           target="_blank"
           class="link validator_link"
         >
-          {{ v }}
+          {{ validator }}
         </BcLink>
         <span>, </span>
       </template>
     </div>
+    <div v-else>
+      {{ cappedValidators.length }} {{ $t('common.validator') }}
```

Comment:
issue: `plural` has to be rendered correctly

```suggestion
      {{ $t('common.validator', {count: cappedValidators.length}) }}
```